websites/P/PERPHeads/dist/metadata.json

```diff
@@ -8,8 +8,8 @@
       "en": "An awesome Garry's Mod community"
     },
     "url": "perpheads.com",
-    "version": "1.0.0",
-    "logo": "https://i.imgur.com/w36WI0Y.png",
+    "version": "1.0.1",
+    "logo": "https://i.imgur.com/RPpka0O.png",
```

Comment:
version should stay 1.0.0 on release

Reply:
also image quality go BRRRRR

Reply:
Hahahahaha, idk what happened there, I'll fix this real quick.

Reply:
I updated it as best as I could and changed the version. I asked a few members of staff whether they had a better logo (quality wise). But seems like because it's such an old logo there's no "better" quality.

Reply:
Might just be better to resize this one https://i.imgur.com/w36WI0Y.png

Reply:
![perp](https://user-images.githubusercontent.com/34792838/105718015-17a23d00-5f21-11eb-8452-a8cfe5910a8f.png)

Reply:
Fixed!